build_pearl.sh

```diff
@@ -0,0 +1,27 @@
+cd "$(dirname "$0")"
+
+BIN_DIR="electron/bins/"
```

Comment:
You need to download the trader binary here. Check [this step](https://github.com/valory-xyz/olas-operate-app/blob/main/.github/workflows/release.yml#L37).

Reply:
I have created a different script for downloading trader binaries, in the same directory

Reply:
Alright, all good. You are missing the header there as well.

Reply:
Okay, I'll add it there as well